docs/source/en/model_doc/dinov2.md

```diff
@@ -109,6 +109,35 @@ predicted_class_idx = logits.argmax(-1).item()
 print("Predicted class:", model.config.id2label[predicted_class_idx])
 ```
 
+The example below shows how to split the output tensor into one embedding for the whole image, i.e. the CLS token, and patch embeddings.
+
+```py
+from transformers import AutoImageProcessor, AutoModel
+from PIL import Image
+import requests
+
+url = 'http://images.cocodataset.org/val2017/000000039769.jpg'
+image = Image.open(requests.get(url, stream=True).raw)
+print(image.height, image.width)  # [480, 640]
+
+processor = AutoImageProcessor.from_pretrained('facebook/dinov2-base')
+model = AutoModel.from_pretrained('facebook/dinov2-base')
+patch_size = model.config.patch_size
+
+inputs = processor(images=image, return_tensors="pt")
+print(inputs.pixel_values.shape)  # [1, 3, 224, 224]
+B, C, H, W = inputs.pixel_values.shape
+h, w = H // patch_size, W // patch_size
```

Comment:
let's use more descriptive naming

batch_size, channels, height, width
num_patches_height, num_patches_width

Reply:
Updated, thanks for the review